README.md

```diff
@@ -93,54 +95,38 @@ Clone this repository to your desired folder:
 
 git clone https://https://github.com/Uchenna-ibute/Leaderboard.git
 
+```sh
+  cd my-folder
+  git clone https://https://github.com/Uchenna-ibute/Leaderboard.git
+  cd Leaderboard
+```
 ### Install
 
 Install this project with:
 
-<!--
-Example command:
-
 ```sh
-  cd my-project
-  gem install
+  npm install
 ```
---->
-
 ### Usage
 
 To run the project, execute the following command:
 
-<!--
-Example command:
-
 ```sh
-  rails server
+  npm run start
 ```
---->
 
 ### Run tests
 
 To run tests, run the following command:
 
-<!--
-Example command:
-
 ```sh
-  bin/rails test test/models/article_test.rb
+npm tests
 ```
---->
 
 ### Deployment
 
-You can deploy this project using:
-
-<!--
-Example:
+You can deploy this project using:[Github pages](https://docs.github.com/en/pages) or any hosting service of your choice.
```

Comment:
- [ ] Kindly mention 1-3 future features to the `Future features` section.
- [ ] Kindly add the API that your app consumes to the README file. ⬇️
![image](https://user-images.githubusercontent.com/33268927/216619639-471a78d5-e6bb-4715-966e-da3c65af3fda.png)